scanner/decisions/0003-scanner-image-names.md

```diff
@@ -0,0 +1,63 @@
+# 0003 - Scanner image names
+
+- **Author(s):** Ross Tannenbaum <rtannenb@redhat.com>
+- **Created:** 2023-08-22
+
+## Status
+
+Accepted.
+
+## Context
+
+The [current version of StackRox Scanner](https://github.com/stackrox/scanner) shipped with each StackRox release is a fork of [Clair v2](https://github.com/quay/clair/tree/v2.1.8) which is versioned and released separately from the rest of StackRox.
+The separate versioning process has proven to cause confusion in the past, so we re-tag each Scanner release with the associated StackRox release.
+For example, the Scanner 2.30.3 release triggered the creation of the following images:
+
+* scanner:2.30.3
+* scanner-slim:2.30.3
+* scanner-db:2.30.3
+* scanner-db-slim:2.30.3
+
+Upon cutting the related StackRox release, 4.1.3, the images were re-tagged as follows:
+
+* `scanner:4.1.3`
+* `scanner-slim:4.1.3`
+* `scanner-db:4.1.3`
+* `scanner-db-slim:4.1.3`
+
+There is an ongoing effort to overhaul the StackRox Scanner to align it more closely with [Clair v4](https://github.com/quay/clair).
+To align with Clair's versioning scheme, we refer to the new StackRox Scanner as StackRox Scanner v4.
+StackRox Scanner v4 is housed in this repository (as opposed to the previous, separate repository) and its release process is more tightly coupled with the StackRox release process.
```

Comment:
If you'd move this ADR to https://github.com/stackrox/architecture-decision-records/ repo, the part of this sentence "housed in this repository" would need to change.

I +1 the suggestion to use ADRs repo, but ultimately it's up to you as there seem to be few other ADRs here already that would need to be moved.

Reply:
For now, I will keep this here, but the Scanner team will discuss this offline. We can bulk move the ADRs from here should we decide to switch repositories